main.tf

```
@@ -726,7 +726,7 @@ resource "aws_elastic_beanstalk_environment" "default" {
   setting {
     namespace = "aws:elasticbeanstalk:command"
     name      = "DeploymentPolicy"
-    value     = var.rolling_update_type == "Immutable" ? "Immutable" : "Rolling"
+    value     = var.deployment_policy != "" ? var.deployment_policy : (var.rolling_update_type == "Immutable" ? "Immutable" : "Rolling")
```

```suggestion
    value     = var.deployment_policy
```

why do we need to mix `rolling_update_type` with `deployment_policy`?
Those are diff concepts not related to each other
It was incorrectly done before, let's fix it now.

---

It was just for backwards compatibility, but yeah they are different, will change it